packages/content/style.css

```diff
@@ -3,6 +3,8 @@
   /* Default highlight color (if no product is specified) */
   --highlight-color: var(--brand);
   --highlight-color-text: var(--brand-link);
+  --visited-link: #5c4ee5;
+  --visited-link-on-hover: #4c40bc;
```

Comment:
It may be appropriate to [add these to `mktg-global-styles`](https://github.com/hashicorp/mktg-global-styles/pull/11), open to thoughts there.